src/assets/i18n/en.json

```diff
@@ -1,8 +1,8 @@
 {
-  "CANCEL": "Cancel",
+  "CANCEL": "I'll save them first",
   "REFRESH": "Refresh",
   "CLOSE": "Close",
-  "OK": "Ok",
+  "OK": "That's okay",
```

Comment:
The same applies here, this change will change all instances of "Ok" to "That's okay", which might look kind of weird in some dialogs.

Reply:
Idem!